scripts/pull_github_pr.sh

```diff
@@ -99,3 +136,5 @@ if [ "$PR_TESTS_STATUS" != "success" ]; then
   echo -e "${ORANGE}\nWARNING:${NC} Some of the tests that ran for this PR were not completed successfully,\n" \
         "please make sure all tests are done successfully before merge this PR.\n"
 fi
+
+check_jenkins_job_status
```

Comment:
Why not do this before the merge even happens? As-is, if the maintainer finds out the job is unstable, they have to undo the merge.

Reply:
Make sense. changed

Reply:
I disagree with @denesb on this :-( I wish we discussed this.

As you can see We already have checks in the script - warning you if the tests didn't pass - and @yaronkaikov put his new check in exactly the same place. The **reason** why these checks are in that place (and last time, it was discussed) - is that maintainers like me sometimes want to try out a commit that they know is not ready. They want to pull it, but not push it. So the script does the pull, but then warns you not to push.

I guess that if you added an option ("--force") to do the operation anyway, then it's fine, but let's move all the tests to one place - it's makes little sense to test Jenkins in one place in a script and Github actions in a different place in the script.

Also, if you didn't do this (I didn't check), please make sure that the error message contains a mention of the option "--force" that you can use to ignore that check - when I need it, I won't remember how that option is called.

Reply:
@nyh I am about to send a follow-up PR with those changes (already talked about it with @denesb )